types/reveal/index.d.ts

```diff
@@ -1,241 +1,1701 @@
-// Type definitions for Reveal 4.2.0
+// Type definitions for Reveal 4.3.1
```

Comment:
```suggestion
// Type definitions for Reveal 4.3
```

See https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library